src/ROUTES.js

```diff
@@ -114,10 +114,12 @@ export default {
     SET_PASSWORD_WITH_VALIDATE_CODE: 'setpassword/:accountID/:validateCode',
     DETAILS: 'details',
     getDetailsRoute: (login) => `details?login=${encodeURIComponent(login)}`,
+    PROFILE: 'a/:accountID',
```

Comment:
👋 Just a heads-up that this has caused a regression in https://github.com/Expensify/App/issues/21969
 Deep linking wasn't working for the new PROFILE route.

 When adding a new route we should also add it to [App/well-known
/apple-app-site-association](https://github.com/Expensify/App/blob/main/.well-known/apple-app-site-association) for iOS
```js
{ "/": "/a/*", "comment": "Profile Page" }
```
And to [AndroidManifest.xml](https://github.com/Expensify/App/blob/main/android/app/src/main/AndroidManifest.xml) for Android 
```js
<data android:scheme="https" android:host="new.expensify.com" android:pathPrefix="/a"/> 
```

Reply:
Oh wow. I had no idea. Can we make that obvious somehow? Like with a checklist item or lint rule or something.

Reply:
@puneetlath, I proposed to add a checklist item in https://expensify.slack.com/archives/C049HHMV9SM/p1690395182200269
Looks like we should add it, I'll do that tomorrow (not sure what the process is just yet, will find it)